src/quartz_mailer/message.cr

```diff
@@ -0,0 +1,98 @@
+require "email"
+
+class Quartz::Message
```

Comment:
This can be a struct instead of a class since is only holding data

Reply:
I prefer we leave this as a class.  Some of this data can stay encapsulated.

Reply:
Yeah.  I still think the `Message` itself represents an object and possible BL, no?

Reply:
For better or for worse, the [EMail::Address](https://github.com/arcage/crystal-email/blob/master/src/email/address.cr#L6) class validates names and addresses.